aptos-move/vm-genesis/src/lib.rs

```diff
@@ -433,6 +436,19 @@ fn initialize_aptos_coin(session: &mut SessionExt<impl MoveResolver>) {
     );
 }
 
+fn initialize_fee_collection_and_distribution(session: &mut SessionExt<impl MoveResolver>) {
+    exec_function(
+        session,
+        "transaction_fee",
+        "initialize_fee_collection_and_distribution",
+        vec![],
+        serialize_values(&vec![
+            MoveValue::Signer(CORE_CODE_ADDRESS),
+            MoveValue::U8(100),
+        ]),
+    );
+}
+
```

Comment:
This ideally should be just calling `init_module`, since when we publish it for the first time, that's what should happen as well. Of course, historical experience suggested that this didn't happen and nearly caused testnet to fall over...